projects/ui/src/shim.cds-core.scss

```
@@ -526,31 +526,58 @@
+  --clr-lightweight-alert-info-icon-color: #{tokens.$cds-alias-status-info};
+  --clr-lightweight-alert-info-color: #{tokens.$cds-alias-typography-color-500};
+
+  // Lightweight Warning type
+  --clr-lightweight-alert-warning-icon-color: #{tokens.$cds-alias-status-warning-dark};
+  --clr-lightweight-alert-warning-color: #{tokens.$cds-alias-typography-color-500};
+
+  // Lightweight Danger type
+  --clr-lightweight-alert-danger-icon-color: #{tokens.$cds-alias-status-danger};
+  --clr-lightweight-alert-danger-color: #{tokens.$cds-alias-typography-color-500};
+
+  // Lightweight Success type
+  --clr-lightweight-alert-success-icon-color: #{tokens.$cds-alias-status-success};
+  --clr-lightweight-alert-success-color: #{tokens.$cds-alias-typography-color-500};
+
   // Alert Spacing
   --clr-alert-padding: #{tokens.$cds-global-space-5};
   --clr-alert-borderradius: #{tokens.$cds-alias-object-border-radius-100};
   --clr-alert-actions-spacing: #{tokens.$cds-global-space-6};
-  --clr-alert-item-margin-bottom: #{tokens.$cds-global-space-2};
-  --clr-alert-line-height: #{tokens.$cds-global-typography-font-size-7};
+  --clr-alert-item-margin-bottom: 0; // ATM - remove
+  --clr-alert-font-size: var(--clr-p5-font-size);
+  --clr-alert-line-height: var(--clr-p5-line-height);
   --clr-alert-app-padding: #{tokens.$cds-global-space-4};
   --clr-alert-icon-wrapper-margin-top: 0; // ATM: delete after token migration
   --clr-alert-items-gap: #{tokens.$cds-global-space-5};
   --clr-alert-items-margin-right: 0; // ATM: delete after token migration
   --clr-alert-icon-margin-left: 0; // ATM: delete after token migration
-  --clr-alert-close-icon-size: #{tokens.$cds-global-space-7};
+  --clr-alert-close-icon-size: #{tokens.$cds-global-space-9};
   --clr-alert-icon-size: #{tokens.$cds-global-space-9};
-
+  --clr-alert-close-button-size: #{tokens.$cds-global-space-11};
+  --clr-alert-lightweight-sm-padding: #{tokens.$cds-global-space-3};
+  --clr-alert-lightweight-font-size: #{tokens.$cds-alias-typography-font-size-4};
+  --clr-alert-lightweight-line-height: var(--clr-p2-line-height);
+  --clr-alert-lightweight-sm-font-size: var(--clr-p6-font-size);
+  --clr-alert-lightweight-sm-line-height: var(--clr-p6-line-height);
+
+  // ATM - use as default
   .alert-app-level {
     --clr-alert-text-flex-grow: 0; // ATM: delete after token migration
+    .alert-items {
+      align-items: center;
+    }
   }
 
   // Alert small
   --clr-alert-sm-line-height: var(--clr-p2-line-height);
   --clr-alert-sm-font-size: #{tokens.$cds-global-typography-font-size-3};
   --clr-alert-sm-letter-spacing: var(--clr-p2-letter-spacing);
   --clr-alert-sm-font-weight: #{tokens.$cds-global-typography-font-weight-regular};
-  --clr-alert-sm-icon-size: #{tokens.$cds-global-space-7};
-  --clr-alert-sm-icon-wrapper-height: #{tokens.$cds-global-space-7};
+  --clr-alert-sm-icon-size: #{tokens.$cds-global-space-9};
+  --clr-alert-sm-icon-wrapper-height: #{tokens.$cds-global-space-9};
```

Why icon sizes are changed to 24px? 
For small alerts icon size is 16px.

```suggestion
  --clr-alert-sm-icon-size: #{tokens.$cds-global-space-7};
  --clr-alert-sm-icon-wrapper-height: #{tokens.$cds-global-space-7};
```